SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -406,6 +406,57 @@ FSpatialNetGUIDCache::FSpatialNetGUIDCache(USpatialNetDriver* InDriver)
 {
 }
 
+typedef TMap<UObject*, uint32> FSubobjectToOffsetMap;
+
+FORCEINLINE FSubobjectToOffsetMap CreateOffsetMapFromActor(Worker_EntityId EntityId, AActor* Actor, const FClassInfo& Info)
+{
+	FSubobjectToOffsetMap SubobjectNameToOffset;
+
+	for (auto& SubobjectInfoPair : Info.SubobjectInfo)
+	{
+		UObject* Subobject = StaticFindObjectFast(UObject::StaticClass(), Actor, SubobjectInfoPair.Value->SubobjectName);
+		uint32 Offset = SubobjectInfoPair.Key;
+
+		if (Subobject != nullptr && Subobject->IsPendingKill() == false && Subobject->IsSupportedForNetworking())
+		{
+			SubobjectNameToOffset.Add(Subobject, Offset);
+		}
+	}
+
+	if (Actor->GetInstanceComponents().Num() > 0)
+	{
+		// Process components attached to this object; this allows us to join up
+		// server- and client-side components added in the level.
+		TArray<UActorComponent*> ActorInstanceComponents = Actor->GetInstanceComponents();
+
+		// These need to be ordered in case there are more than one component of the same type, or
+		// we may end up with wrong component instances having associations between them.
+		ActorInstanceComponents.Sort([](const UActorComponent& Lhs, const UActorComponent& Rhs) -> bool {
+			return Lhs.GetName().Compare(Rhs.GetName()) < 0;
+		});
+
+		for (UActorComponent* DynamicComponent : ActorInstanceComponents)
```

Comment:
```suggestion
		for (const UActorComponent* DynamicComponent : ActorInstanceComponents)
```

Reply:
No can do - we populate the output map with non-const subobject pointers, as they are potentially modifiable by calling code.